### PR TITLE
Fixed issue on generate path

### PR DESCRIPTION
**Pull request by naveen965**

### CODE REVIEW

File: Assets/PathCreator/Examples/Scripts/GeneratePathExample.cs [hunk 1]
```diff
@@ -30,13 +32,17 @@ void Start ()
                 BezierPath bezierPath = new BezierPath (waypoints, closedLoop, PathSpace.xyz);
                 GetComponent<PathCreator> ().bezierPath = bezierPath;
             }
+
+            Selection.activeGameObject = path.gameObject;
```
Comment (naveen965) [inline Assets/PathCreator/Examples/Scripts/GeneratePathExample.cs:36]: In this line correct for editor version

Comment (senaveen5) [reply Assets/PathCreator/Examples/Scripts/GeneratePathExample.cs:36]: Oh, can you fix it

Comment (naveen965) [reply Assets/PathCreator/Examples/Scripts/GeneratePathExample.cs:36]: sure

**Review by senaveen5 — APPROVED** [review]
Approved

**Pull request merged**